Sources/SwiftFusion/Geometry/Rot2.swift

```diff
@@ -48,23 +48,25 @@ extension Rot2: CustomDebugStringConvertible {
   }
 }
 
+/// Group actions.
 extension Rot2 {
   /// Returns the result of acting `self` on `v`.
   @differentiable
-  func rotate(_ v: Vector2) -> Vector2 {
-    Vector2(c * v.x - s * v.y, s * v.x + c * v.y)
+  public func rotate(_ v: Vector2) -> Vector2 {
+    coordinate.rotate(v)
```

Comment:
Just a comment: @marcrasi I still think it's weird we have this coordinate business :-/ It's like we double up everything? I know I can probably be convinced again, but just conveying my gut feeling here that it might look weird to outsiders. @tuwuhs what was your impression of this? We're genuinely interested in your first impression here.

Reply:
I think this coordinate thing does make sense for rotations, since we can have more than one way to implement a rotation. For instance `Rot3` can be implemented either as a rotation matrix or a quaternion. For rigid transforms I agree that it seems like a lot of code duplications with no clear advantage (at least in my limited experience working with Swift and Lie groups). What is the main rationale behind separating the coordinate type? Is it some subtle Swifty thing? Or perhaps something to do with differentiability?